### PR TITLE
Bumped version, v1.0.0 comp and notification

### PR DESCRIPTION
**Pull request by DieterKoblenz**
Added a very simple notification handler. You can add an action card and put text in it.

### CODE REVIEW

File: lib/system.js [hunk 1]
```diff
@@ -29,6 +30,16 @@ function onFlowRestartMCU (callback) {
   }).catch(callback)
 }
 
+function onNotification (callback, args) {
+		//console.log('Message: ',args.noti);
```
Comment (matjaz) [inline lib/system.js]: please remove

File: lib/system.js [hunk 2]
```diff
@@ -29,6 +30,16 @@ function onFlowRestartMCU (callback) {
...
+		Homey.manager('notifications').createNotification({
+			excerpt: args.noti
+		}, ( err, notification ) => { console.log('sent',err,notification); });
```
Comment (matjaz) [inline lib/system.js]: I would remove console.log and move callback here.

Comment (DieterKoblenz) [reply lib/system.js]: Sure, however what do you mean with "move callback here"?

Comment (matjaz) [reply lib/system.js]: inside of `( err, notification ) =>` function.

**Review by DieterKoblenz — COMMENTED** [review]
Like this

**Comment by matjaz** [comment]
https://github.com/matjaz/info.matjaz.candy/commit/f4e806d5771237a2a6889dabd0e4815771eb49a6

**Comment by matjaz** [comment]
thanks!

**Pull request merged**